scss/_typography.scss

```diff
@@ -7,15 +7,25 @@ $base-font-family: "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, "
 // Base typography
 // ----------------------------------------------
 
-h1,
+h1 {
+	margin: 0;
+  padding: 0;
+  font-weight: 700;	
+}
 h2,
 h3,
 h4,
-h5,
 h6 {
   margin: 0;
```

Comment:
h1-h6 have the same base margins, so you would want to cascade your changes below.

```css
h1,
h2,
h3,
h4,
h5,
h6 {
  margin: 0;
  padding: 0;
  font-weight: 400;
}

h1 {
  font-weight: 700;
}

h5 {
  font-weight: normal;
  text-transform: uppercase;
}